recipes/sip/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    pythonw configure.py
+else
+    python configure.py
```

Comment:
Good call. I think `--sysroot` is enough because `bindir` is `$PREFIX/bin` (which is not our default on Windows).